yb-voyager/cmd/root.go

```diff
@@ -159,14 +159,26 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 
 		// Log the flag values set from the config file
 		for _, f := range overrides {
+			// If the config key is source.db-password, target.db-password, or source-replica.db-password, mask the value
+			if f.ConfigKey == "source.db-password" || f.ConfigKey == "target.db-password" || f.ConfigKey == "source-replica.db-password" {
```

Comment:
nit: let's create lists for the config-keys/env-vars to obfuscate in config.go and use them here.